apps/api/src/exception-filter.ts

```diff
@@ -96,6 +80,41 @@ export class AllExceptionsFilter implements ExceptionFilter {
       message: `Internal server error, contact support and provide them with the errorId`,
     };
   }
+  private handleCommandValidation(exception: CommandValidationException): ResponseMetadata {
+    const DO_NOT_TRACK_CLASSES = [ProcessBulkTriggerCommand.name];
+
+    let uuid: string | undefined;
+    if (!DO_NOT_TRACK_CLASSES.includes(exception.className)) {
+      uuid = this.getUuid(exception);
+      this.logError(uuid, exception);
```

Comment:
we added last week log for all 4XX in buildResponseBody the same place that 5XX logs. so i think we can remove this one.

Reply:
this is not a common practice, why do we log all 4XX errors?

Reply:
And did you notice it doesn't go to sentry? is it on purpose? I would like to talk about this and make sure we got all the usecases covered

Reply:
all resolved offline